pom.xml

```diff
@@ -580,6 +580,11 @@
       <artifactId>tika-parsers</artifactId>
       <version>1.12</version>
     </dependency>
+    <dependency>
+      <groupId>org.rogach</groupId>
+      <artifactId>scallop_2.12</artifactId>
```

Comment:
FWIW, bespin is on `2.11`: https://github.com/lintool/bespin/blob/master/pom.xml#L244

Reply:
As noted below, this removed the error I was encountering.